docs/site/Model.md

```diff
@@ -221,6 +408,8 @@ class MyUserModel extends Entity {
 
 ### Property Decorator
 
+<!-- Property decorator can reuse lb3 docs -->
```

Comment:
Yes please!

Ideally, we should copy information from https://loopback.io/doc/en/lb3/Model-definition-JSON-file.html#properties.

I am not sure if that's in scope of this pull request though.

I see that we have the following text after the example code snippet, thus I think we are good for now.

> The complete list of valid attributes for property definitions can be found in LoopBack 3's [Model definition section](https://loopback.io/doc/en/lb3/Model-definition-JSON-file.html#properties).

Reply:
Thanks for these comments!